modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/learn_message/LearnMessage.tsx

```diff
@@ -117,13 +119,24 @@ export default function LearnMessage({
 	if (learnMessageObject.url) {
 		return (
 			<ClayLink
+				aria-label={sub(
+					Liferay.Language.get('x-opens-new-window'),
+					learnMessageObject.message
+				)}
 				className={getCN('learn-message', className)}
 				href={learnMessageObject.url}
 				rel="noopener noreferrer"
 				target="_blank"
 				{...otherProps}
 			>
 				{learnMessageObject.message}
+
+				<ClayButtonWithIcon
```

Comment:
It doesn't feel right to have a button within a link. Could we just have `<ClayIcon>` here?